src/cephadm/cephadmlib/container_types.py

```diff
@@ -8,28 +8,8 @@
 from typing import Dict, List, Optional, Any, Union, Tuple, Iterable, cast
 
 from .call_wrappers import call, call_throws, CallVerbosity
-from .constants import (
-    DEFAULT_TIMEOUT,
-    # default container images
-    DEFAULT_ALERT_MANAGER_IMAGE,
-    DEFAULT_GRAFANA_IMAGE,
-    DEFAULT_LOKI_IMAGE,
-    DEFAULT_NODE_EXPORTER_IMAGE,
-    DEFAULT_PROMETHEUS_IMAGE,
-    DEFAULT_PROMTAIL_IMAGE,
-    DEFAULT_HAPROXY_IMAGE,
-    DEFAULT_KEEPALIVED_IMAGE,
-    DEFAULT_NVMEOF_IMAGE,
-    DEFAULT_SNMP_GATEWAY_IMAGE,
-    DEFAULT_ELASTICSEARCH_IMAGE,
-    DEFAULT_JAEGER_COLLECTOR_IMAGE,
-    DEFAULT_JAEGER_AGENT_IMAGE,
-    DEFAULT_JAEGER_QUERY_IMAGE,
-    DEFAULT_SMB_IMAGE,
-    DEFAULT_SMBMETRICS_IMAGE,
-    DEFAULT_NGINX_IMAGE,
-    DEFAULT_OAUTH2_PROXY_IMAGE,
-)
+from .constants import DEFAULT_TIMEOUT
+from ceph.cephadm.images import DefaultImages
```

Comment:
just to make you aware, I messed with these imports in the PR to move these to a common location (missed it originally as I didn't rebase the PR before testing) and those changes will conflict with these changes. You can either rebase your work on top of what I've since done with that PR, or wait until I merge it and solve conflicts. Up to you.